docs/index.rst

```diff
@@ -0,0 +1,106 @@
+Shotgun Hiero Export API reference, |release|
+###################################################
+
+
+The ``tk-hiero-export`` app adds custom Shotgun export processors to the Hiero/Nuke Studio export framework.
+
+Via custom export processors, the user can create and update entities in the current Project in Shotgun.
+
+During the export process a number things can happen:
+
+- Status of associated Shot entities can be updated
```

Comment:
i would ❤️ if there was a flow chart diagram showing the process and included all the hooks. I think that would be a perfect overview for TDs to understand what is possible. happy to help putting this together if you need help.

Reply:
Agreed, we can chat offline about this.